pkgs/flox-bash/default.nix

```diff
@@ -87,7 +87,7 @@ in
   stdenv.mkDerivation rec {
     pname = "flox-bash";
     version = "${czToml.tool.commitizen.version}-${inputs.flox-floxpkgs.lib.getRev self}";
-    src = "${self}/flox-bash";
+    src = ( toString self ) + "/flox-bash";
```

Comment:
Just curious .. why? It's quite idiomatic in nixpkgs to see files in packages referred to by way of `${pkgname}/foo/bar` strings - what does that break that `toString` fixes?

Reply:
It avoids redundantly copying `self` to the nix store.

This is an unfortunate UX issue in the implementation details of string interpolation on paths in `nix`.

In order to avoid copying `/nix/store/aaaaa...` to `/nix/store/bbbb...`, then expanding to `/nix/store/bbbb.../flox-bash` you want to use either `( toString self ) + "/flox-bash"` or `self.outPath + "/flox-bash"`.

It's an unfortunate issue that's been griped about for in upstream `nix` issues a lot. There's intended to be a fix eventually, but for now the recommendation is this uglier form that doesn't use string interpolation on `path` types.